src/web/components/Caption.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+
+import { Section } from '@frontend/web/components/Section';
+import { Caption } from '@frontend/web/components/Caption';
+
+export default {
+    component: Caption,
+    title: 'Components/Caption',
+};
+
+/**
+    display: Display;
+    designType: DesignType;
+    captionText?: string;
+    pillar: Pillar;
+    padCaption?: boolean;
+    credit?: string;
+    displayCredit?: boolean;
+    shouldLimitWidth?: boolean;
+    isOverlayed?: boolean; // Not tested here as this option only works in the context of the ImageComponent
+ */
```

Comment:
to remove

Reply:
Why? I often paste the props in the stories as comments so the reader has an immediate reference to help understand the scope of testing

Reply:
dont you get that from typescript anyway by simply hover over the component?
ex:
![Screenshot 2020-06-17 at 10 13 33](https://user-images.githubusercontent.com/8831403/84879548-44d16780-b083-11ea-834b-c7d2317c411a.png)

Reply:
Sure, there are helper tooltips. I just find it easier to see it on the page rather then having to reveal it by moving my mouse. I guess the real question is: why not?

Reply:
> why not?
sure ^^